### ISSUE TITLE
🗺 world map symbols are not rendered correctly

### ISSUE THREAD

**Issue by socketpair** [xtermjs/xterm.js#2987]
See https://gitlab.gnome.org/GNOME/vte/-/issues/265

exactly the same bug.

**Comment by jerch** [comment]
@socketpair Would be nice to get some more info on that, like unicode version it was specified that way. xterm.js can only handle Unicode v6 by default, and v11 with the `unicode-v11` addon.

Btw a "cell width" of 4 of a no-go for most terminal emulators currently. There is simply no way to address more than 2 cells by a single codepoint. Sadly most "modern" Unicode inventions are incompatible with terminals' monospaced idea of rendering things. Even worse - the Unicode consortium doesnt care at all, thus there is no clear patch path either.

To sum up things - this codepoint very unlikely to work across terminals and fonts settings soon.
